hapi/hedera-protobufs/services/node_create.proto

```diff
@@ -137,4 +137,29 @@ message NodeCreateTransactionBody {
      * This field is REQUIRED and MUST NOT be set to an empty `KeyList`.
      */
     proto.Key admin_key = 7;
+
+    /**
+    * An ALT_BN128 elliptic curve public encryption key (also controlled <br/>
+     * by the node operator) specific to this node's TSS operations.
+     * <p>
+     * The elliptic curve type may change in the future. For example, <br/>
```

Comment:
No reason to put breaks every line, we _want_ the lines to merge in the documentation.
```suggestion
     * The elliptic curve type MAY change in the future. For example,
```